modules/contributing-quick-starts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="contributing-quick-starts_{context}"]
+= Contributing quick starts
+
+In {product-title} 4.7, a quick start custom resource is added. This allows Operators and administrators to contribute new quick starts to the cluster. Typically, quick starts for Operators are created by the Operator itself after the Operator is installed. In a few cases, out-of-the-box quick starts that guide administrators through the process of installing an Operator are available. These quick starts must be created before Operator installation through Operator Hub.
```

Comment:
I'd remove the rest of the paragraph starting at "Typically, quick starts for Operators are created by the Operator itself..." That's more of an internal implementation detail and shouldn't matter to cluster admins.